### DISCUSSION TITLE
Docker container web iu

### DISCUSSION THREAD

**Discussion by jupitert25** [engageub/InternetIncome#76] [Ideas]
It would be great if you add Docker container web iu to manager on browser other PC

**Comment by engageub** [comment]
Hello,
Could you please elaborate what you meant by that.
Please note that adding any extra features will increase resources and running multiple instances will sum up the extra feature on all containers.
The intention is only to make money as much as possible using limited resources and not to play games. 

Thank you

**Reply by engageub** [reply]
You can install DOCKER WEB UI: PORTAINER to see the list of containers running. The script does what is required once started and manual intervention is not required. It restarts automatically when the PC is restarted.
